src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -457,6 +464,125 @@ public RealJenkinsRule prepareHomeLazily(boolean prepareHomeLazily) {
         return this;
     }
 
+    /**
+     * Use {@link #withFIPSEnabled(Path)} with default value of {@code target/dependency}
+     * @return
+     */
+    public RealJenkinsRule withFIPSEnabled() {
+        return withFIPSEnabled(Path.of("target", "dependency"));
+    }
+
+    /**
+     * Enable FIPS mode assuming all FIPS BouncyCastle jars (bc-fips.jar, bctls-fips.jar and bcpkix-fips.jar) are in the provided directory.
+     * This will set some System properties such :
+     * <ul>
+     *   <li>-Xbootclasspath with FIPS BouncyCastle jars</li>
+     *   <li>java.security.properties to a new temp file configuring the security provider to FIPS BouncyCastle</li>
+     *   <li>-Dsecurity.overridePropertiesFile=true</li>
+     *   <li>-Djavax.net.ssl.trustStoreType=PKCS12</li>
+     *   <li>-Djenkins.security.FIPS140.COMPLIANCE=true</li>
+     *   <li>-Dcom.redhat.fips=false</li>
+     * </ul>
+     * @param fipsLibrariesPath path to directory with FIPS BouncyCastle jars
+     * @return
```

Comment:
@return on its own is an invalid javadoc comment and will produce warnings, please add some text after it or remove it.

```suggestion
     * @return This real Jenkins rule.
```